Monika After Story/game/script-songs.rpy

```diff
@@ -1767,6 +1767,42 @@ label mas_song_we_have_all_the_time_in_the_world_long:
     m 1ekbla "{i}{cps=18}~Only love~{/cps}{/i}"
     return
 
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_songs_database,
+            eventlabel="mas_song_night_fever",
+            category=[store.mas_songs.TYPE_SHORT],
+            prompt="Luka Luka Night Fever",
+            random=True,
+            aff_range=(mas_aff.ENAMORED,None)
+        ),
+        code="SNG"
+    )
+
+label mas_song_night_fever:
+    m  3hub "{i}~Hitting edges and crawling on by~ {/i}"
+    m  3eub "{i}~Reading pixels and every new line~{/i}"
+    m  3rusdlb "{i}~Sitting tired of internet yet{/i}{w=0.5}{nw}"
+    extend 1hub "{i} staring blankly at the screen ahead~{/i}"
+    m  1fub "{i}~Did you know that I’m gonna be free?~ {/i}"
```

Comment:
```suggestion
    m  1eub "{i}~Did you know that I’m gonna be free?~ {/i}"
```

Reply:
Ow, I wanted to use `1fub`... I need to think about it.